### PR TITLE
Made changes to Artsy contact and updated date

### PR DESCRIPTION
**Pull request by ystarr**
Reviewed and updated last two sections of API Terms

### CODE REVIEW

**Comment by dblock** [comment]
@orta I'm helping Yayoi with this, for now don't merge.

File: app/views/content/terms.md [hunk 1]
```diff
@@ -275,10 +275,8 @@ To the fullest extent permitted by applicable law, you agree to indemnify, defen
...
 
-20.3 Artsy is located at: 401 Broadway, 25th Floor, New York, NY 10013, U.S.
+20.3 Artsy is located at: 401 Broadway, 26th Floor, New York, NY 10013, U.S.
 
 ##21. Effective Date
 
-These Terms are effective September 9, 2014.
-
-Last Modified: September 9, 2014
+Last Modified: February 20, 2017.
```
Comment (dblock) [inline app/views/content/terms.md]: Thanks for fixing the date here!

**Comment by orta** [comment]
😍

**Comment by ystarr** [comment]
Still reviewing these terms -- trying to get them shorter and slightly more friendly.  Major changes will include: (i) allowing production; (ii) stressing that this is for non-commercial use only; and (iii) making clear that the Artwork Images and resulting info being provided is public domain.

Hope to finish my review before week's end and will ping you again then!

File: app/views/content/terms.md [hunk 2]
```diff
@@ -28,39 +28,31 @@ The following definitions are used in these Terms:
...
+2.1 You may use the API only if you can form a binding contract with us, and only as permitted by law. You are not permitted to use the API if you are a person under the age of eighteen (18). If you are accepting these Terms and using the API on behalf of a company, organization, or other legal entity, you represent and warrant that you are authorized to bind that entity to these Terms, in which case the terms “you” and “your” in these Terms will refer to that entity. 
 
-2.5 You agree to register for an Artsy account and provide all required information. You agree to provide only true and accurate information in connection with your account, and to update all such information as necessary to keep it accurate and current. 
+2.2 Your use of the API is also subject to our general [Terms of Use](https://artsy.net/terms) and [Privacy Policy](https://artsy.net/privacy) as amended by us from time to time. By accessing or using the Services in any manner, you agree to be bound by these terms and conditions.
```
Comment (dblock) [inline app/views/content/terms.md]: Fix this link, add `www.`, so https://www.artsy.net/terms. It redirects, but useful for SEO.

**Comment by ystarr** [comment]
@orta @dblock -- as I continue to think this through, practical question for you experts both:

1.  Currently, do we have any restrictions on what API is released?  (I ask this b/c I understand from a biz perspective, provenance and valuation data, for e.g., is our special sauce, and if possible, I'd like to not have our project be the open door to release this info.)

2.  To cover the above concern, I think that releasing Public Domain images for non-commercial use will help a lot.  Just to understand the bucket of stuff that we intend to release based upon this concept, if we decide to just release public domain images + metadata:

a.  How many works is this (even approximately)?

b.  How many artists is this (even approximately)?

c.  Are these hi-res or low-res images?

d.  Does it include values of genome data?

e.  Will the metadata include transaction/ownership (for e.g., gallery transactions or historical provenance) -- if so, how many records?

Look forward to your thoughts and, as always, thanks so much for dealing with me the newbie!
Y

**Comment by dblock** [comment]
@ystarr 

Instead of limiting what we release in terms, we make sure that the API behaves in the same way as the terms describe. So if we release metadata + public domain images, that's what the API returns by default.

_a. How many works is this?_

26024 works

_b. How many artists is this (even approximately)?_

The API currently returns all artists with their metadata, doesn't restrict to artists that have public domain works, which is 70776. If we decide to only release artists with public domain works that would be 4321 artists.

_c. Are these hi-res or low-res images?_

It varies wildly depending on the provider.

_d. Does it include values of genome data?_

We have never released genome data. We don't plan to currently.

_e. Will the metadata include transaction/ownership (for e.g., gallery transactions or historical provenance) -- if so, how many records?_

No.

**Comment by ystarr** [comment]
@dblock 

Thank you so much for the update.  Makes sense.

Final confirmation and question -- artists with public domain works is 4321 artists.  The resulting image numbers would be 26,024 works or smtg less?

Thanks
Y

**Comment by dblock** [comment]
Correct @ystarr, we have 26,024 pbublic domain works by 4321 artists. Note that number changes all the time :)

**Comment by ystarr** [comment]
Great and thanks @dblock .  More soon.

**Comment by ystarr** [comment]
@dblock @orta AT LAST, I've gone in and finally completed my review of this.  Thank you HUGELY for your patience.  I am now ready for you to review and commit my changes to the API!

Again, for full clarity, at this point in time, the API should return public domain works, no genome data, no commercial records (for e.g., gallery transactions, ownership or historical provenance), and no commercial inventory.  And the API Terms are for non-commercial use by the user only.

Thank you to you both!!!

File: app/views/content/terms.md [hunk 3]
```diff
@@ -76,77 +68,59 @@ The following definitions are used in these Terms:
...
 
 ##6. Attribution and Publicity
 
-6.1 We may include attribution requirements applicable to the use of API Content in Documentation or otherwise make such requirements available to you. We may also post policies or guidelines applicable to the use of our Brand Features on the Services or otherwise make such policies or guidelines available to you. You agree to comply with all such requirements, policies, and guidelines, as amended by us from time to time.
+6.1 In Your Application, you agree to post the following notice: “This application uses the Artsy API but is not endorsed or certified by Artsy.” The words "Artsy API" in this notice must hyperlink to the developers section of the Artsy website at: www.developers.artsy.net. We may include additional attribution requirements applicable to the use of API Content in Documentation or otherwise make such requirements available to you. You agree to comply with all such requirements, as amended by us from time to time.
```
Comment (dblock) [inline app/views/content/terms.md]: The www is extra here and doesn't exist/work.

File: app/views/content/terms.md [hunk 4]
```diff
@@ -76,77 +68,59 @@ The following definitions are used in these Terms:
...
 
-6.2 Somewhere in Your Application, you will post the following notice: “This application uses the Artsy API but is not endorsed or certified by Artsy.” The words "Artsy API" in this notice must hyperlink to the developers section of the Artsy website at: www.developers.artsy.net. 
+6.2 If you use an Artwork Image in Your Application, in addition to the notice required in section 6.1 above, you will post the following notice prominently in Your Application: "This application features images from the Artsy API." The words "Artsy API" in this notice must hyperlink to the developers section of the Artsy website at: www.developers.artsy.net. We may request that you change the size, location and/or language of this notice in Your Application at any time at our sole discretion, and you agree to comply with any such requests as soon as reasonably practicable.
```
Comment (dblock) [inline app/views/content/terms.md]: Same here, `www` extra.

**Comment by dblock** [comment]
This is great, see my comment on two typos, otherwise ship it!

**Comment by ystarr** [comment]
Thanks to all for their review...naive question -- what happens next?

**Comment by ystarr** [comment]
Made those two changes — ready for Orta to kindly review and ship.  Thank you!

**Comment by orta** [comment]
👍 This looks good to go - shipping it, thanks for the PR @ystarr 

🐰 💅

**Pull request merged**